dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/AnnotationLiteralsTest.war/resources/annotationLiterals.xhtml

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<!--
+    Copyright (c) 2022 IBM Corporation and others.
```

Comment:
2023?

Reply:
Yes, some of these were written in 2022.